contracts/ActiveBridgeSetBlockRelay.sol

```diff
@@ -268,6 +268,19 @@ contract ActiveBridgeSetBlockRelay is BlockRelayInterface {
     return blocks[_blockHash].relayerAddress;
   }
 
+  /// @dev Pay the block reward to the relayer in case it has not been paid before
+  /// @param _blockHash Hash of the block header
+  function payRelayer(uint256 _blockHash) external payable override {
```

Comment:
```suggestion
  function payRelayer(uint256) external payable override {
```

Reply:
this change should remove the warnings

Reply:
![image](https://user-images.githubusercontent.com/44392885/108061233-4f039580-7058-11eb-9cc8-41d141ec92c2.png)